.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "aicsimageio/metadata/czi_to_ome"]
+	path = aicsimageio/metadata/czi_to_ome
+	url = https://github.com/AllenCellModeling/czi-to-ome-xslt.git
```

Comment:
Are there any other clients of this new repo?  Who else would use it?

Reply:
I have a feeling Zeiss may want to use it in their libCZI library? But no one besides us currently using it.

Reply:
It was a leading question --> why is it separate?

Reply:
Yeah it's very green if it gains traction I think it could become of value to the community but it needs to do some growing first.

Reply:
In my mind there are two reasons it's separate. 1) The underlying tool is language agnostic and isn't bound to `aicsimageio`. 2) I personally feel that people will be more willing to make a PR to that repo to add metadata to those sheets than make a PR to `aicsimageio` as it has become a rather large beast of a repo imo